lib/layout.js

```diff
@@ -215,8 +215,10 @@ El.calcSize = (el) => (
 
 El.calcScrollSize = (el) => (
   el === window
-    ? { w: el.scrollX, h: el.scrollY }
-    : { w: el.scrollLeft, h: el.scrollTop }
+   ? {
+    w: el.scrollX || el.pageXOffset || document.documentElement.scrollLeft,
+    h: el.scrollY || el.pageYOffset || document.documentElement.scrollTop
```

Comment:
Why would you reach out to the `html` element? And assuming there is a valid reason I'm not sure I like this approach. This was intentionally a pure function before [1].

[1] Excepting the "type" check on `el`